cover.sh

```diff
@@ -0,0 +1,9 @@
+go test -v ./... -coverprofile=coverage.txt -covermode=atomic
```

Comment:
Can we put this script in `./scripts`  :)

Reply:
sure

Reply:
done.